.github/workflows/auto-respond.yml

```
@@ -13,13 +16,34 @@ jobs:
-            const issue = context.issue;
+            // 忽略机器人提交（例如 dependabot）
+            const sender = context.payload.issue?.user?.login;
+            if (!sender || sender.endsWith('[bot]')) {
+              core.info(`Skipping auto-response for bot user: ${sender}`);
+              return;
+            }
+
+            const { owner, repo } = context.repo;
+            const issue_number = context.issue.number;
             const body = `感谢你提交 issue！为帮助我们更快定位并复现问题/需求，请补充下列信息（如适用）：
 
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 29: did not find expected key

(syntax-check)